documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -11,6 +11,11 @@
 
 ## Version 10.3.0
 
+### Resolved issues
+
+* (IDETECT-4610) - Improved [detect_product_short] Airgap for Gradle Creation script to remove unwanted JARs from being included.
+* (IDETECT-4611) - Improved [detect_product_short] Airgap for Gradle Creation script to remove reference to Integration Common library that is no longer a dependency.
```

Comment:
* (IDETECT-4611) - Updated [detect_product_short] air gap for Gradle creation script to remove reference to Integration Common library that is no longer a dependency.

Reply:
Done, thanks.